pkg/pdapi/pdapi_test.go

```diff
@@ -634,6 +634,98 @@ func TestDeleteStore(t *testing.T) {
 	}
 }
 
+func TestGetEvictLeaderSchedulersForStores(t *testing.T) {
+	g := NewGomegaWithT(t)
+
+	expectHTTPMethod := "GET"
+	expectURLPath := fmt.Sprintf("/%s", schedulersPrefix)
+
+	type testcase struct {
+		name string
+
+		storeIDs []uint64
+		resp     []byte
+		expect   func(schedulers map[uint64]string, err error)
+	}
+
+	cases := []testcase{
+		{
+			name:     "all store have evict scheduler",
+			storeIDs: []uint64{1, 7, 8},
+			resp: []byte(`
+[
+	"evict-leader-scheduler-1",	
+	"evict-leader-scheduler-7",	
+	"evict-leader-scheduler-8",	
+	"evict-leader-scheduler-9"
+]
+			`),
+			expect: func(schedulers map[uint64]string, err error) {
+				g.Expect(err).To(Succeed())
+				storeIDs := []uint64{1, 7, 8}
+				g.Expect(schedulers).To(HaveLen(len(storeIDs)))
+				for storeID, scheduler := range schedulers {
+					g.Expect(storeIDs).To(ContainElement(storeID), "store id isn't in storeIDs")
+					g.Expect(scheduler).To(Equal(getLeaderEvictSchedulerStr(storeID)), "scheduler is't corresponding to store id")
+				}
+			},
+		},
+		{
+			name:     "some store have evict scheduler",
+			storeIDs: []uint64{1, 7, 10},
+			resp: []byte(`
+[
+	"evict-leader-scheduler-1",	
+	"evict-leader-scheduler-7",	
+	"evict-leader-scheduler-8",	
+	"evict-leader-scheduler-9"
+]
+			`),
+			expect: func(schedulers map[uint64]string, err error) {
+				g.Expect(err).To(Succeed())
+				storeIDs := []uint64{1, 7, 10}
+				for storeID, scheduler := range schedulers {
+					g.Expect(storeIDs).To(ContainElement(storeID), "store id isn't in storeIDs")
```

Comment:
Need to check the len(schedulers).

Reply:
`schedulers` is a superset of B in this case, so the length of `schedulers` isn't important.